tests/unit-tests/admin/test-class-sensei-setup-wizard.php

```diff
@@ -339,7 +346,9 @@ public function testRenderWizardPage_WhenRendered_ClearsRedirectOption() {
 		update_option( 'sensei_activation_redirect', 1 );
 
 		// Act.
+		ob_start();
 		Sensei()->setup_wizard->render_wizard_page();
+		ob_end_clean();
```

Comment:
This prevents an output when running the tests.

Reply:
Special thanks for this one 😅